src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -1377,11 +1378,10 @@ def migrate_database(self) -> None:
                     SQLModel.metadata.create_all  # type: ignore[arg-type]
                 )
             with Session(self.engine) as session:
-                session.add(
-                    IdentitySchema(
-                        id=str(GlobalConfiguration().user_id).replace("-", "")
-                    )
+                deployment_id = os.getenv(
+                    ENV_ZENML_DEPLOYMENT_ID, GlobalConfiguration().user_id
```

Comment:
When are we setting this ID?

Reply:
When creating cloud tenants to have their ID align with the tenant ID

Reply:
Got it! Thanks.